dashboard/src/components/AppView/AppView.tsx

```diff
@@ -19,7 +19,7 @@ import ResourceTable from "./ResourceTable";
 export interface IAppViewProps {
   namespace: string;
   releaseName: string;
-  app: IRelease;
+  app?: IRelease;
```

Comment:
Ditto

Reply:
same reason, app will be undefined while fetching